File: testdata/dupe/typescript/sample.ts
// Fixture used by internal/tokenizers/typescript's tests.

export function processOrder(items: number[]): number {
  let total = 0;
  for (const item of items) {
    if (item > 0 && item < 1000) {
      total += item;
    }
  }
  return total;
}

export function processInvoice(items: number[]): number {
  let total = 0;
  for (const item of items) {
    if (item > 0 && item < 1000) {
      total += item;
    }
  }
  return total;
}

export function unrelated(): string {
  return "nothing shared here";
}

File: testdata/dupe/python/sample.py
# Fixture used by internal/tokenizers/python's tests.


def process_order(items):
    total = 0
    for item in items:
        if item > 0 and item < 1000:
            total += item
    return total


def process_invoice(items):
    total = 0
    for item in items:
        if item > 0 and item < 1000:
            total += item
    return total


def unrelated():
    return "nothing shared here"
